learn/update_and_migration/versioning.md

```diff
@@ -0,0 +1,37 @@
+# Versioning policy
+
+Meilisearch releases contain performance improvements, bug fixes, and new features, but may not always maintain full compatibility between Meilisearch versions.
+
+This article describes the system behind Meilisearch's release versions and how they relate to SDK and documentation versions.
+
+## Engine versioning
```

Comment:
This is a lot of text. Without getting too much into the content—which strikes me as a bit confusing—we need bullet points or a table here to make it more readable, plus sub-sections detailing the answers to specific questions users might have (e.g. how frequently will the core engine update? which versions will be supported long-term? how do we handle inter-version compatibility?). Break it down to make the page more scannable.

I find the way SEMVER is presented in the integration versioning policy to be pretty good, although the language is probably more technical than we would like:

> 🔥 Given a version number MAJOR.MINOR.PATCH, increment the:
>
> 1. MAJOR version when you make incompatible API changes
> 2. MINOR version when you add functionality in a backwards compatible manner
> 3. PATCH version when you make backwards compatible bug fixes